Source/DataSources/PlaneGeometryUpdater.js

```diff
@@ -344,27 +334,19 @@ function createPrimitiveMatrix(
     -distance,
     scratchTranslation
   );
-  translation = Matrix4.multiplyByPoint(transform, translation, translation);
-
-  var transformedNormal = Matrix4.multiplyByPointAsVector(
-    transform,
-    normal,
-    scratchNormal
-  );
-  Cartesian3.normalize(transformedNormal, transformedNormal);
 
-  var up = ellipsoid.geodeticSurfaceNormal(translation, scratchAxis2);
+  var up = Cartesian3.clone(Cartesian3.UNIT_Z, scratchUp);
   if (
     CesiumMath.equalsEpsilon(
-      Math.abs(Cartesian3.dot(up, transformedNormal)),
+      Math.abs(Cartesian3.dot(up, normal)),
       1.0,
       CesiumMath.EPSILON8
     )
   ) {
-    up = Cartesian3.clone(Cartesian3.UNIT_Z, up);
+    up = Cartesian3.clone(Cartesian3.UNIT_Y, up);
     if (
```

Comment:
I know this isn't code you added, but I think we can safely delete this section. There's no way the initial up guess is aligned to both the Z and Y axes, so setting to `UNIT_X` will never happen.